src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/validator/AppointmentValidatorTest.kt

```diff
@@ -79,7 +89,7 @@ internal class AppointmentValidatorTest {
       }
 
       @Test
-      fun `empty address for non nps office appointment throws validation error`() {
+      fun `empty address for a non nps delius appointment throws validation error`() {
```

Comment:
In the user/domain language we still call this NPS location; can we keep the previous way?